docs/source/configuration/settings-reference.md

```diff
@@ -200,8 +200,8 @@ controlpanels
 
     The group can be one of the default groups 'General', 'Content', 'Security', 'Add-on Configuration', 'Users and Groups' or a custom group.
 
-filterControlPanelsSchema
-    A schema factory for a control panel. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.
+unwantedControlPanelsFields
+    Control panels fields that are not used on the Volto. It is used internally, to tweak the schemas provided by the controlpanel endpoint, to make them fit for Volto.
```

Comment:
This also might need a version number tweak at release.

````suggestion
    Control panels fields that are not used on the Volto.
    It is used internally to tweak the schemas provided by the `controlpanel` endpoint, to make them fit for Volto.

    ```{versionchanged} Volto 16.20.8
    `filterControlPanelsSchema` was renamed to `unwantedControlPanelsFields` in Volto 16.20.8.
    ```
````